colors/spaceduck.vim

```diff
@@ -79,12 +81,12 @@ elseif &background == 'light'
 
   hi SpaceduckForeground guifg=#ecf0c1 ctermfg=255 guibg=NONE ctermbg=NONE gui=NONE cterm=NONE
 
-  let s:palette.bg = s:palette.cream
+  let s:palette.bg = s:palette.white
```

Comment:
@jinh0 I used the `cream` color for the background since it's less aggressive on the eyes and that's also what @pineapplegiant asked for here: https://github.com/pineapplegiant/spaceduck/issues/24#issuecomment-822202420.

Reply:
Oh yeah that was an accident, I'll revert it back!

Reply:
Alright no problem @jinh0.

Reply:
Hey @Nargonath , we have a Discord server for spaceduck where we discuss new changes to the colorscheme. It would help speed up communication. You're welcome to join! https://discord.gg/tW7YfWjM

Reply:
Alright, thanks @jinh0. I'll join the discord server.